contracts/cryptoblades.sol

```diff
@@ -85,7 +84,6 @@ contract CryptoBlades is Initializable, AccessControlUpgradeable {
     }
 
     // config vars
-    uint characterLimit;
```

Comment:
State variables can never be removed due to proxy contract upgradeability. Instead, add `// UNUSED; KEPT FOR UPGRADEABILITY PROXY COMPATIBILITY` as a note above it.